### ISSUE TITLE
Existing or new void worlds not loading

### ISSUE THREAD

**Issue by jermangarc** [XCompWiz/Mystcraft-Issues#138]
I have a player that has a void world that was working for a long time. All of a sudden the world stopped loading upon teleporting there and freezes the server. It's only happening in void worlds. We also have RFTools installed and the same exact issue is happening with that mod as well. I'm getting a log file ready and starting a new world without RFTools to see if it happens without it. Doing vice versa with Mystcraft.

https://github.com/McJty/RFTools/issues/382

**Comment by XCompWiz** [comment]
Sounds like it's an issue with another mod and void worlds. This, unfortunately, could be any mod, not just the mods that create the worlds.

Without a log I can say nothing about what mod is doing it. Chances are, neither Mystcraft nor RFTools are directly involved beyond creating the dimensions.

**Comment by jermangarc** [comment]
I agree, I uninstalled Mystcraft and tried making a void age with RFTools and it did the same thing. I do have JustAnotherSpawner which I use to control mob spawning, but uninstalled that one as well with no resolution to the problem. So with 322 mods to go, it'll be a long week ahead, lol. That's the problem when you make a mega modpack, lol. It was working fine up until a few updates ago, so I'll have to narrow it down to that list. Thanks for your help anyhow, it's a start. I'll go ahead and close this since I found Mystcraft not to be the problem. I'll comment my findings when I find it, so it can help others' that might have the same issue.
